core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -113,8 +115,10 @@ export class Topic {
     this._pageTitleFragmentForWeb = pageTitleFragmentForWeb;
   }
 
-  // ---- Instance methods ----
-  getId(): string {
+  // Some methods have either string or null return value,
+  // because when we create interstitial topic object
+  // their fields get null value.
```

Comment:
```suggestion
  // Returns 'null' when the topic is not yet saved on the backend.
```

And add same comment to the functions below.

Reply:
Done